content/messaging/customer-chat-api/v3.2/rtm-reference/index.mdx

```diff
@@ -1148,6 +1156,7 @@ It returns [summaries](#chat-summaries) of the chats a Customer participated in.
   "type": "response",
   "success": true,
   "payload": {
+    "next_page_id": "MTUxNzM5ODEzMTQ5Ng==", // optional
```

Comment:
Soon, we'll be getting rid of "JSON comments". What do you say we delete the comment and make a note in the Response table instead?